src/AlgebraicGeometry/ToricVarieties/cohomCalg/VanishingSets/attributes.jl

```diff
@@ -65,14 +65,15 @@ Normal, non-affine, smooth, projective, gorenstein, fano, 2-dimensional toric va
 
 julia> vs = vanishing_sets(dP1)
 3-element Vector{ToricVanishingSet}:
- Toric vanishing set for cohomology index 0
- Toric vanishing set for cohomology index 1
- Toric vanishing set for cohomology index 2
+ Toric vanishing set for cohomology indices [0]
+ Toric vanishing set for cohomology indices [1]
+ Toric vanishing set for cohomology indices [2]
 
-julia> cohomology_index(vs[3])
-2
+julia> cohomology_indices(vs[3])
+1-element Vector{Int64}:
+ 2
 ```
 """
-function cohomology_index(tvs::ToricVanishingSet)
-    return tvs.i::Int
+function cohomology_indices(tvs::ToricVanishingSet)
+    return tvs.cis::Vector{T} where {T <: IntegerUnion}
```

Comment:
```suggestion
    return tvs.cis
```
Don't see why it should be necessary.

Reply:
I specified the return value, because I believed that this allows `Julia` to optimize the performance. Is this wrong? Or is the return value inferred differently here?

Reply:
The field of `tvs.cis` is defined as `Vector{<: IntegerUnion}` already in the definition of the structure. So julia will always know what type `.cis` is.

On the other hand, since this is not a concrete type, it is not that useful. Accessing `.cis` the compiler still does not know what it is exactly (could be for example `Vector{Int}`, `Vector{Union{Int, ZZRingElem}}` or `Vector{ZZRingElem}`) , so the performance will be bad.